src/core/Form/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import {
+  Checkbox as CompCheckbox,
+  CheckboxProps as CompCheckboxProps,
+  CheckboxInputProps as CompCheckboxInputProps,
+} from '../../../components/Form/Checkbox';
+import { baseStyles } from './Checkbox.baseStyles';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import classnames from 'classnames';
+
+const baseClassName = 'fi-checkbox';
+
+const checkboxClassNames = {
+  error: `${baseClassName}--error`,
+  checked: `${baseClassName}--checked`,
+  large: `${baseClassName}--large`,
+};
+
+export interface CheckboxProps extends CompCheckboxProps, TokensProp {}
+export interface CheckboxInputProps
+  extends CompCheckboxInputProps,
+    TokensProp {}
```

Comment:
This indentation looks weird, is this really what our prettier config now produces?

Reply:
I guess it does since it passes all linting.

Reply:
I replied to this ages ago and even saw the answer here for a long while. Where the hell did it disappear? But yeah, that's what prettier does here.